syntax = "proto3";

service LoggingService {
  rpc LogMessage (LogRequest) returns (LogResponse);
}

message LogRequest {
  string uuid = 1;
  string msg = 2;
}

message LogResponse {
  string status = 1;
}
